.golangci.yml

```diff
@@ -1,18 +1,31 @@
-run:
-  build-tags:
-  - e2e
-  skip-dirs:
-  - vendor
-  - pkg/client
-  timeout: 5m
 linters-settings:
   errcheck:
     exclude: .errcheck.txt
 linters:
+  disable-all: true
   enable:
```

Comment:
so that's explicitly enable only those following linters, which is i think good, but just making sure that's what you want